File: 프로그래머스/0/181901. 배열 만들기 1/배열 만들기 1.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n, int k) {
    int m = 1;
    vector<int> v;
    for(int i = k; i <= n; i = k * (++m)) {
        v.push_back(i);
    }
    
    return v;
}

File: 프로그래머스/0/181901. 배열 만들기 1/배열 만들기 1.java
import java.util.*;

class Solution {
    public ArrayList<Integer> solution(int n, int k) {
        ArrayList<Integer> answer = new ArrayList<>();
        
        for(int i = k; i <= n; i += k) {
            answer.add(i);
        }
        
        return answer;
    }
}